.github/workflows/lychee-cron.yml

```diff
@@ -0,0 +1,54 @@
+name: Check Links In Public Directory
+
+on:
+  pull_request:
```

Comment:
having this here allows for the initial action to trigger, once this is approved, im going to remove this so manual trigger and scheduled cron task are the triggers for the task.